cdap-ui/app/wrangler/components/Wrangler/Wrangler.less

```diff
@@ -225,6 +256,24 @@
       border-top: 2px solid #f39c12;
     }
 
+    .header-content-container {
+      display: inline-block;
+      width: 100%;
+
+      .header-text {
+        cursor: pointer;
+        display: inline-block;
+        width: calc(~"100% - 40px");
```

Comment:
Not needed with autoprefixer: https://github.com/postcss/autoprefixer/issues/575